buyer-app/buyer-app/src/components/buyer/DataOrderCreate/DataOrderCreate.js

```diff
@@ -95,15 +94,15 @@ class DataOrderCreate extends Component {
       selectedBuyer
     } = this.state;
 
-    const selectedAudience = audience.map(filter => {
-      return {
-        filter: filter.variable,
-        values: [filter.value]
-      };
-    });
+    // const selectedAudience = audience.map(filter => {
```

Comment:
Rather than commenting things I would just remove it.

Reply:
Makes sense, I'm always afraid of removing functionality... like is doing something anyway 🤷‍♂